creating script for com_028_rw1 (#214)
resource-watch/data-pre-processing: com_028_rw1_effect_of_ag_prices_on_commodity_prices/README.md

## Effect of Agricultural Policies on Commodity Prices Dataset Pre-processing

> This file describes the data pre-processing that was done to [the Nominal Rate of Protection](http://www.ag-incentives.org/indicator/nominal-rate-protection) for [display on Resource Watch]({link to dataset's metadata page on Resource Watch}).

Reviewer: need to add RW link

> The data source provided the dataset as one csv file and one README file. The csv file was reformated to upload to Carto. We converted the years in the 'year' column to datetime objects and store them in a new column 'datetime'. We removed the column 'notes' since it contains an index without descriptions. The column names were converted to lowercase before we uploaded the data to Carto. 

Reviewer 2: ```suggestion
The data source provided the dataset as one csv file. Years in the 'year' column were converted to datetime objects and stored in a new column 'datetime'. The 'notes' column was removed since it only contains indexes instead of actual data. The column names were converted to lowercase before we uploaded the data to Carto. 
```

Please see the [Python script](https://github.com/resource-watch/data-pre-processing/blob/master/com_028_rw1_effect_of_ag_prices_on_commodity_prices/com_028_rw1_effect_of_ag_prices_on_commodity_prices_processing.py) for more details on this processing.

You can view the processed Effect of Agricultural Policies on Commodity Prices dataset [on Resource Watch]({link to dataset's metadata page on Resource Watch}).

You can also download the original dataset [directly through Resource Watch](https://wri-public-data.s3.amazonaws.com/resourcewatch/com_028_rw1_effect_of_ag_prices_on_commodity_prices.zip), or [from the source website](http://www.ag-incentives.org/indicator/nominal-rate-protection).

> ###### Note: This dataset processing was done by [Weiqi Zhou](https://www.wri.org/profile/weiqi-zhou), and QC'd by [{name}]({link to WRI bio page}).

Reviewer 2: ```suggestion
###### Note: This dataset processing was done by [Weiqi Zhou](https://www.wri.org/profile/weiqi-zhou), and QC'd by [{Yujing Wu}]({https://www.wri.org/profile/yujing-wu}).
```
